### PR TITLE
Added "/dd4hep::cm" to the DTGeometryBuilder Class 

### PR DESCRIPTION
**Pull request by slomeo**
**PR description:**

Added "/ ddhep::cm" to the DD4hep part of the DTGeometryBuilder Class in order to simplify a possible future unit transition 

**PR validation:**

1) validation by "cmsRun Geometry/RPCGeometryBuilder/test/python/validateDTGeometry_cfg.py" and "cmsRun Geometry/RPCGeometryBuilder/test/python/validateDTGeometryDDD_cfg.py" 

See attached picture (only one histo for DD4hep)

+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

 2) validation by printouts (myLog.log), for both DD & DD4Hep, created by the validation scripts used above (for example please see, below, the parameters check for the 574914560 (Chamber) detid, 574922752 (SLayer) detid, 574923776 (Layer) detid ):

//DDD

(0) DTGeometryBuilder - DDD 
(1) detId: 574914560, par[0]: 1090 mm, par[1]: 1255.5 mm, par[2]: 181 mm
(2) detId: 574922752, par[0]: 1063.2 mm, par[1]: 1255.5 mm, par[2]: 26.75 mm
(3) detId: 574923776, par[0]: 1029.65 mm, par[1]: 1199 mm, par[2]: 5.75 mm
(4) detId: 574923776 ,wireLenght in ddd, (wpar[1] in dd4hep): 117.4 cm, firstWire: 1

//DD4Hep

(0) DTGeometryBuilder - DD4Hep 
(1) detId: 574914560, par[0]: 109 cm, par[1]: 125.55 cm, par[2]: 18.1 cm
(2) detId: 574922752, par[0]: 106.32 cm, par[1]: 125.55 cm, par[2]: 2.675 cm
(3) detId: 574923776, par[0]: 102.965 cm, par[1]: 119.9 cm, par[2]: 0.575 cm
(4) detId: 574923776, wpar[1]: 117.4 cm, firstWire: 1  
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

3) validation by "runTheMatrix.py -l 25202.1" :

25202.1_TTbar_13+TTbar_13+DIGIUP15APVSimu_PU25+RECOUP15_PU25+HARVESTUP15_PU25 Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Mon Dec 14 18:32:47 2020-date Mon Dec 14 18:10:42 2020; exit: 0 0 0 0  1 1 1 1 tests passed, 0 0 0 0 failed

**if this PR is a backport please specify the original PR and why you need to backport that PR:**

nothing special

Schermata 2020-12-14 alle 17 16 53

### CODE REVIEW

**Comment by cvuosalo** [comment]
@cmsbuild please test

File: Geometry/DTGeometryBuilder/plugins/dd4hep/DTGeometryBuilder.cc [hunk 1]
```diff
@@ -50,6 +51,8 @@ using namespace std;
 using namespace cms;
 
 void DTGeometryBuilder::buildGeometry(DDFilteredView& fview, DTGeometry& geom, const MuonGeometryConstants& num) const {
+  edm::LogVerbatim("DTGeometryBuilder") << "(0) DTGeometryBuilder - DD4Hep ";
+
```
Comment (cvuosalo) [inline Geometry/DTGeometryBuilder/plugins/dd4hep/DTGeometryBuilder.cc:55]: It looks like line 89 also needs conversion:
```
 new Plane(Surface::PositionType(tr[0], tr[1], tr[2]),
```
I believe these are positions in cm.

File: Geometry/DTGeometryBuilder/src/DTGeometryBuilderFromDD4Hep.cc [hunk 2]
```diff
@@ -49,6 +50,8 @@ using namespace std;
 void DTGeometryBuilderFromDD4Hep::buildGeometry(cms::DDFilteredView& fview,
                                                 DTGeometry& geom,
                                                 const MuonGeometryConstants& num) const {
+  edm::LogVerbatim("DTGeometryBuilder") << "(0) DTGeometryBuilder - DD4Hep ";
+
```
Comment (cvuosalo) [inline Geometry/DTGeometryBuilder/src/DTGeometryBuilderFromDD4Hep.cc:54]: Same translation problem here, same line 89.

Comment (slomeo) [reply Geometry/DTGeometryBuilder/src/DTGeometryBuilderFromDD4Hep.cc:54]: @cvuosalo : done

**Comment by civanch** [comment]
please test

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**